src/snowcli/cli/project/schema.py

```diff
@@ -106,7 +106,7 @@ def as_fully_optional(self) -> RelaxedMap:
 streamlit_schema = RelaxedMap(
     {
         "name": Str(),
-        "stage": Str(),
+        Optional("stage", default="streamlit"): Str(),
```

Comment:
Please update docstrings for streamlit deploy with information that default stage can be used. This will help users use `--help` to understand the behaviour.

Reply:
@sfc-gh-turbaszek Currently it says "If you don't provide any stage name then 'streamlit' stage will be used." -- do you think that needs to be changed?

Reply:
That ^ was really me in disguise, by the way :)

Reply:
Oh, that's fine, I thought we have changed it!